lib/cn/csched.c

```diff
@@ -12,110 +12,61 @@
 #include <hse_ikvdb/ikvdb.h>
 #include <hse_ikvdb/cn.h>
 
-#include "csched_ops.h"
-#include "csched_noop.h"
 #include "csched_sp3.h"
 
-struct tbkt;
-
 merr_t
 csched_create(
-    enum csched_policy   policy,
     struct mpool *       ds,
     struct kvdb_rparams *rp,
     const char *         kvdb_alias,
     struct kvdb_health * health,
     struct csched **     handle)
 {
-    merr_t             err;
-    struct csched_ops *cs;
-
-    assert(rp);
-    assert(kvdb_alias);
-    assert(handle);
-
-    err = merr(EINVAL);
+    assert(rp && kvdb_alias && handle);
```

Comment:
May want to use INVARIANT. Only reason I would keep as 3 separate calls is because now it isn't obvious which parameter was NULL.

Reply:
Done.